python/ply_io.py

```diff
@@ -29,15 +29,22 @@ def read_ply(fp, newline=None):
             if 'element face' in line:
                 raise Exception('.ply appears to be a mesh')
             if 'end_header' in line: break
-    
+
+        for j in range(len(prop)):
```

Comment:
Instead of adding this section I would change line 28 to prop.append(line.split()[2].replace('scalar_', ''))